tests/test_orchestrated_responses_stream.py

```
@@ -0,0 +1,279 @@
+            {"role": "system", "content": "Answer in Korean."},
+            {"role": "user", "content": "Research, implement, and verify the design."},
+        ],
+        mode="conduct",
+    )
+
+    assert observed_system_messages
+    assert all("Caller instructions:\nAnswer in Korean." in message for message in observed_system_messages)
+
+
+def test_duplicate_workflow_roles_close_each_reasoning_summary_part() -> None:
+    token = "duplicate_role_stream_token"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "token"

(S105)